README.md

```diff
@@ -50,7 +67,9 @@ Suppose you had the following form in your HTML file. Use jQuery to write a sing
 ```
 
 ### Question 7
-This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code? 
+This code looks like it works, but when you run it, you see that the `UserApp.add_all_users()` function executes but `console.log` displays `undefined`. What's wrong with the code?
+
+-You got me.  I'm stumped.
```

Comment:
The problem is that `UserApp.add_all_users(data);` needs to be inside of the `.done()` callback in order to have access to the `data` argument.